File: app/src/main/java/com/ipartha/t2s/roomdb/ConsumerMenuDao.java
package com.ipartha.t2s.roomdb;

import androidx.room.Insert;
import androidx.room.Query;

import org.jetbrains.annotations.NotNull;

import java.util.List;

public interface ConsumerMenuDao {
    @Insert
    void insertAll(@NotNull List var1);

    @Query("SELECT * FROM consumer_menu")
    @NotNull
    List getAll();
}

File: app/src/main/java/com/ipartha/t2s/roomdb/ConsumerMenuDao.kt
package com.ipartha.t2s.roomdb

import androidx.lifecycle.LiveData
import androidx.room.Dao
import androidx.room.Insert
import androidx.room.Query
import com.ipartha.t2s.mvvm.ConsumerMenu

@Dao
interface ConsumerMenuDao {
    @Insert
    fun insertAll(menuOptionList : List<ConsumerMenu>)

    @Query("SELECT * FROM "+DBConstants.CONSUMER_MENU_TABLE_NAME)
    fun getAll() : LiveData<List<ConsumerMenu>>
}